python/ray/serve/constants.py

```diff
@@ -13,6 +13,9 @@
 #: HTTP Port
 DEFAULT_HTTP_PORT = 8000
 
+#: Controller checkpoint path
+DEFAULT_CHECKPOINT_PATH = "ray://"
```

Comment:
not to use `gcs` because:
1. gcs is internal to ray architecture and unfamiliar with user
2. gcs can be confused with google cloud storage.